includes/class-sensei-grading.php

```diff
@@ -1131,11 +1131,34 @@ public static function get_graded_lessons_sum() {
 
 	}
 
+	/**
+	 * Get average grade of all lessons graded
+	 *
+	 * @since 4.2.0
+	 * @return double $graded_lesson_avg
+	 */
+	public static function get_graded_lessons_avg() {
+		$grade_args        = array(
+			'type'     => 'sensei_lesson_status',
+			'status'   => 'any',
+			'meta_key' => 'grade',  // phpcs:ignore Slow query ok.
```

Comment:
Are we sure that this slow query is OK? How slow is it with 100,000+ users?

Reply:
I have removed this altogether since I need to do the proper query anyway, I will not use the activity here.